src/node_http2.cc

```diff
@@ -55,10 +55,27 @@ enum Http2OptionsIndex {
   IDX_OPTIONS_FLAGS
 };
 
+static uint32_t http2_padding_buffer[3];
+static uint32_t http2_options_buffer[IDX_OPTIONS_FLAGS + 1];
+static uint32_t http2_settings_buffer[IDX_SETTINGS_COUNT + 1];
+static double http2_session_state_buffer[IDX_SESSION_STATE_COUNT];
+static double http2_stream_state_buffer[IDX_STREAM_STATE_COUNT];
+
+static const size_t http2_options_buffer_byte_length =
+    sizeof(http2_options_buffer) * (IDX_OPTIONS_FLAGS + 1);
+static const size_t http2_settings_buffer_byte_length =
+    sizeof(http2_settings_buffer) * (IDX_SETTINGS_COUNT + 1);
+static const size_t http2_padding_buffer_byte_length =
+    sizeof(http2_padding_buffer) * 3;
+static const size_t http2_stream_state_buffer_byte_length =
+    sizeof(http2_stream_state_buffer) * IDX_STREAM_STATE_COUNT;
+static const size_t http2_session_state_buffer_byte_length =
+    sizeof(http2_session_state_buffer) * IDX_SESSION_STATE_COUNT;
```

Comment:
What I meant was that Node aims to support multiple independent `Environment` instances; I’m not worried about embedders messing with the contents, I’m thinking of race conditions when embedders have per-thread Node instances and more than one of them is trying to access these values at a time.